src/app/clusters/boolean-state-server/CodegenIntegration.h

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/boolean-state-server/boolean-state-cluster.h>
+
+namespace chip::app::Clusters::BooleanState {
+
+CHIP_ERROR SetStateValue(EndpointId endpointId, const Attributes::StateValue::TypeInfo::Type & stateValue,
```

Comment:
this perserves previous API but it does not look like a great API ... how about having a `GetClusterInstance(EndpointId)` instead and make use of that?